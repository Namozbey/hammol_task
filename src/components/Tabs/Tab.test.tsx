import React from "react";
import { render } from "@testing-library/react";
import { Tab } from ".";

describe("Testing Tab", () => {
  it("Snapshots", () => {
    const { asFragment } = render(<Tab label="Doc" value="doc" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
