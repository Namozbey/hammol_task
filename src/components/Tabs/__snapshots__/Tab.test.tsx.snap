// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Tab Snapshots 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ joHccK"
    value="doc"
  >
    <p
      class="sc-gsnTZi ZIjmp"
    >
      Doc
    </p>
  </div>
</DocumentFragment>
`;
